addons/mail/static/src/discuss/channel_invitation.xml

```diff
@@ -2,23 +2,23 @@
 <templates xml:space="preserve">
 
     <t t-name="discuss.ChannelInvitation" owl="1">
-        <div class="o-discuss-ChannelInvitation d-flex flex-column" t-att-class="{ 'o-discuss-ChannelInvitation-has-size-constraints': props.hasSizeConstraints }">
+        <div class="o-discuss-ChannelInvitation bg-view rounded d-flex flex-column" t-att-class="{ 'o-discuss-ChannelInvitation-has-size-constraints': props.hasSizeConstraints, 'border border-top-0': !env.inChatWindow }">
```

Comment:
maybe remove `border-top-0`. At first, I thought the popover overlays the header but I feel like it changes with the zoom-in ratio and the monitor you have. So we can just make it as usual.

Reply:
Ok to remove `border-top-0`. I think this was used because of the arrow, but with `rounded` it's ugly regardless.
Would be nice to not see arrow, but popover doesn't allow to customize that.